apps/common/behaviours/email.js

```diff
@@ -6,8 +6,8 @@ const StatsD = require('hot-shots');
 const client = new StatsD();
 const Model = require('../models/email');
 const config = require('../../../config');
-
 const { customAlphabet } = require('nanoid');
+const APPLICATION_TYPE = require('../fields/application-type.js')['application-type'].options;
```

Comment:
is there a reason for this to be in capitals?

Reply:
I have amended this to camel case and made the name a bit clearer

Reply:
👍